source/WindowScroller/WindowScroller.jest.js

```diff
@@ -241,44 +253,51 @@ describe('WindowScroller', () => {
       let windowScroller
 
       render(getMarkup({
-        headerElements: <div id='header' style={{ height: 100 }}></div>,
+        headerElements: <div id='header' style={{ height: 100, width: 100 }}></div>,
         ref: (ref) => {
           windowScroller = ref
         }
       }))
 
       expect(windowScroller._positionFromTop).toBe(100)
+      expect(windowScroller._positionFromLeft).toBe(100)
 
       mockGetBoundingClientRectForHeader({
-        height: 200
+        height: 200,
+        width: 200
```

Comment:
I usually like to use different values for width/height or scrollLeft/scrollTop to avoid possibly missing a bug where I transpose the 2 internally.